sequentially-metrics/src/main/scala/com/evolutiongaming/concurrent/MeteredSequentiallyAsync.scala

```diff
@@ -6,13 +6,21 @@ import scala.concurrent.Future
 
 object MeteredSequentiallyAsync {
 
+  type Factory[K] = String => SequentiallyAsync[K]
+
   def apply[K](sequentially: SequentiallyAsync[K],
                name: String,
                sequentiallyMetrics: SequentiallyMetrics.Factory,
   ): SequentiallyAsync[K] = {
     apply(sequentially, sequentiallyMetrics(name))
   }
 
+  def apply[K](sequentially: => SequentiallyAsync[K],
```

Comment:
I believe `apply` method in companion object should have return type of companion class

Reply:
you are right